x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/GetDataFrameTransformsStatsAction.java

```diff
@@ -153,16 +155,33 @@ public Response(List<DataFrameTransformStateAndStats> transformsStateAndStats, L
         public Response(StreamInput in) throws IOException {
             super(in);
             transformsStateAndStats = in.readList(DataFrameTransformStateAndStats::new);
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
How does this work when you don't provide a concrete version (e.g. 7.3.0) but "CURRENT"?

Reply:
@przemekwitek `CURRENT` is whatever the latest version is in that branch. Honestly, I probably could make it 7.3.0 as there are no BWC tests for Dataframe transforms yet.

Reply:
@przemekwitek my personal workflow when doing things that could break BWC is:

* Set the serialization code to refer to `CURRENT` for the change
* Merge that PR
* Prepare for backport with appropriate version branch and change `CURRENT` to that version
* Mute the appropriate BWC tests in Master
* Merge the backport
* Unmute and change from `CURRENT` to real version in Master PR. 

The reason I do it this way is so the code gets exercised through the BWC tests throughout the process. If I muted BWC in this PR, I could potentially miss other BWC test failures.

Reply:
Thanks for the explanation Ben!